include/ck/tensor_operation/gpu/grid/gridwise_gemm_xdlops_v2r4.hpp

```diff
@@ -344,6 +300,14 @@ struct GridwiseGemm_bk0mk1_bk0nk1_mn_xdlops_v2r4
         const auto block_work_idx =
             c_block_cluster_adaptor.CalculateBottomIndex(make_multi_index(get_block_1d_id()));
 
+        // if(!block_2_ctile_map.ValidCTileIndex(
```

Comment:
remove

Reply:
Good catch. I enabled valid c-tile check instead for consistency.